include/taskolib/Sequence.h

```diff
@@ -239,6 +256,20 @@ class Sequence
      */
     ConstIterator erase(ConstIterator first, ConstIterator last);
 
+    /**
+     * The sequence contains a common setup script that is shared by all of its steps. It
```

Comment:
This is fine as a detailed description, but the first sentence is not a good summary of the function.
The way we have configured Doxygen, the first sentence (up to the first dot) is treated as the brief description of the documented entity. In some places, this is the only line of text that is shown, and therefore "This sequence contains a common setup script that is shared by all of its steps" is not helpful. It is better to start with something very short like "Set the step setup script." – note that by convention, this uses active voice, not passive like "This function sets...".